docs/error-reporting/project-setup/file-attachments.md

```diff
@@ -0,0 +1,96 @@
+
+If your integration path to Backtrace does not provide a way to upload minidump files and attachments to Backtrace, you can use our generic HTTP submission methods. You have two options:
+
+### Multipart POST
+A submission where the test.json attachment is included with the initial crash submission. You will need to modify the following to successfully submit to a project within your account:
+1. {Path_to_your_file}: Location of file containing crash data to send.
+1. {Path_to_your_attachment}: Location of file to be attached to the crash.
+1. {universe}: First part of the URL used to access your Backtrace account.
+1. {error-token}: An error token for the project you want to submit crash data to.
+
+```curl
+curl -v -F "upload_file=@<Path_to_your_file>/test.json" -H "Accept: application/json" -F "attachment_test.json=@<Path_to_your_file>/test.json; type=application/json" "https://<universe>.sp.backtrace.io/api/post?token=<error-token>&format=json"
+```
+
+### Attach a File to an Existing Crash Report
+For this method you will need the _rxid value assigned to a submitted crash report.  This value will be returned after a submitting a properly formatted request. The first curl command is the submission of the crash report. The second is the attachment of a file to the first.
+1. {Path_to_your_file}: Location of file containing crash data to send.
+1. {universe}: First part of the URL used to access your Backtrace account.
+1. {error-token}: An error token for the project you want to submit crash data to.
+
+```curl
+curl -d <Path_to_your_file> -H "Accept: application/json" "https://<universe>.sp.backtrace.io/api/post?token=<error-token>&format=json"
+```
+
+A properly formatted submission should return a response like:
+```curl
+{"response":"ok","_rxid":"56000000-8be7-5806-0000-000000000000"}
+```
+
+To attach a file to this object you will need to copy the _rxid returned.
+1. &object=<_rxid>: Value returned from first HTTP submission.
+1. &attachment_name=<Path_to_your_attachment>: Location of file to be attached to the crash.
+```curl
+curl -v --data-binary "upload_file=@<your_file_path>/test.json" -H "Expect: gzip" -H "Content-Type:application/json" "https://<univers>.sp.backtrace.io/api/post?token=<error-token>&format=json&object=<_rxid>&attachment_name=<Path_to_your_attachment>"
+```
+
+A properly formatted upload submission should return a response like:
+```curl
+{"response":"ok","_rxid":"ce000000-0000-0000-0000-000000000000","attachment_name": "test.json","attachment_id": "28","object":"ce"}
+```
+
+The response should tell you that the file was attached to the specified object within your Backtrace project. You will still get the 200/ok response but the _rxid is going to look a bit different.  As there was not a crash submitted it will not send a new unique id. This time it is actually sending the id attribute/Error identifier within the debug view used to identify the specific crash report.
```

Comment:
```suggestion
The response should tell you that the file was attached to the specified object within your Backtrace project. You will still get the 200/ok response, but the _rxid is going to look a bit different. As there was not a crash submitted, it will not send a new unique id. This time it is actually sending the id attribute/Error identifier within the debug view used to identify the specific crash report.
```